css/style.css

```diff
@@ -90,7 +92,7 @@ a{
     justify-content: center;
    
 }
-
+/*something random*/
```

Comment:
what's the use of adding something random

Reply:
Think it was just some random test, I forgot now lol